Source/WebKit/UIProcess/Extensions/WebExtension.cpp

```diff
@@ -90,6 +107,11 @@ double WebExtension::manifestVersion()
     return 0;
 }
 
+bool WebExtension::hasRequestedPermission(String permission) const
+{
+    return m_permissions.contains(permission);
```

Comment:
This is missing `populatePermissionsPropertiesIfNeeded()`, which I just realize was missing in the original impl!

Reply:
hasRequestedPermission is const, but populate* is not.

Reply:
Hmm, try dropping `const`? Or do a dance with `mutable` or `const_cast`?

Reply:
Or at minimum file a bug and add a FIXME comment so we don't forget to fix this. Since it isn't new, that is fine for now.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=280101